server/controllers/auth.js

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable no-console */
+/* eslint-disable no-unreachable */
```

Comment:
Remove the eslint-disable lines that are no longer needed

Reply:
Modified...

Reply:
:)